tests/Doctrine/Tests/DBAL/Types/SimpleArrayTest.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Doctrine\Tests\DBAL\Types;
+
+use Doctrine\DBAL\Platforms\AbstractPlatform;
+use Doctrine\DBAL\Types\Type;
+use Doctrine\Tests\DBAL\Mocks\MockPlatform;
+use const E_ALL;
+use const E_STRICT;
+use function error_reporting;
+use function serialize;
+
+class SimpleArrayTest extends \Doctrine\Tests\DbalTestCase
+{
+    /**
+     * @var AbstractPlatform
+     */
+    protected $_platform;
```

Comment:
`private` by default please

Reply:
The underscore naming should also be used (our CS checks will likely catch this)